crates/ruff/src/rules/flake8_pytest_style/rules/parametrize.rs

```diff
@@ -558,19 +564,31 @@ fn check_duplicates(checker: &mut Checker, values: &Expr) {
     else {
         return;
     };
+    if elts.len() < 2 {
```

Comment:
In general I'd probably prefer something like `let [mut prev, ..] = elts else { return; };`.

Reply:
It better couples the condition to the access, whereas here, we have the condition up here, then the potentially-unsafe `elts[0]` below.